main.go

```diff
@@ -1284,7 +1284,7 @@ func waitForSync(ctx context.Context, base chan dbtypes.SyncResult, aux chan dbt
 }
 
 func connectNodeRPC(cfg *config, ntfnHandlers *rpcclient.NotificationHandlers) (*rpcclient.Client, semver.Semver, error) {
-	return rpcutils.ConnectNodeRPC(cfg.DcrdServ, cfg.DcrdUser, cfg.DcrdPass,
+	return rpcutils.ConnectNodeRPC(cfg.DcrdServ, "rpcuser", "rpcpass",
```

Comment:
@njirap: Probably an accident, but these can't be hard-coded. Let me know if you are experiencing configuration issues I can help with.

Reply:
yes @buck54321 it was an accident to push the change, otherwise this is how I have it in my local setup the `cfg.DcrdUser` and `cfg.DcrdPass` weren't working for me from the beginning so this was my was of hacking it

Reply:
If you haven't already, check out the README section on setting up the configuration file.

https://github.com/decred/dcrdata#creating-the-dcrdata-configuration-file

It's easiest just to run `--help` to see where your system's default config file location is. 

The dcrd RPC authentication is set by setting `dcrduser` and `dcrdpass` in the configuration file.